lib/account.js

```diff
@@ -118,6 +120,16 @@ export const acceptDM = (dm, inboxUser) => {
 
 }
 
+export const writeMedia = (filename, attachment) => {
```

Comment:
We should do the processing inside the main writeMediaFile/readMediaFile functions in storage.js to make it transparent here.